pkg/reconciler.v1/common/README.md

```diff
@@ -6,19 +6,19 @@ To use the reconciler, following methods must be overridden according to the API
 
 ```go
 // GetJob returns the job that matches the request
-func (r *KubeflowJobReconciler) GetJob(ctx context.Context, req ctrl.Request) (client.Object, error)
+func (r *JobReconciler) GetJob(ctx context.Context, req ctrl.Request) (client.Object, error)
```

Comment:
What about something like Training or TrainingJob since we are always dealing with model trainings here. Job seems very generic.

Reply:
`TrainingJob` sounds good to me. As the common library is designed for all kinds of controllers, I would suggest renaming the `GenericJob` to `TrainingJob` in the pull request I mentioned above for tf-operator (https://github.com/kubeflow/tf-operator/pull/1398). What do you think?

Reply:
SGTM